docs/mesos/README.md

```diff
@@ -0,0 +1,34 @@
+# Mesos with Calico Networking
+Calico provides networking for your Mesos cluster. These guides will get you started.
```

Comment:
Could use a little friendlier welcome here, mirroring the calico-mesos repo:

**Calico provides IP-Per-Container Networking for your Mesos Cluster.** The following collection of guides will walk you through the steps necessary to get up and running.